src/agora/agora.cpp

```diff
@@ -807,41 +808,25 @@ void Agora::print_per_frame_done(PrintType print_type, size_t frame_id)
     if (!kDebugPrintPerFrameDone)
         return;
     switch (print_type) {
-    case (PrintType::kPacketRX): {
-        size_t prev_frame_slot = (frame_id - 1) % TASK_BUFFER_FRAME_NUM;
-        printf("Main thread: received all packets in frame: %zu in %.2f us."
-               " Demul: %zu done. RX in prev frame: %zu of %zu.\n",
-            frame_id,
-            stats->master_get_delta_us(
-                TsType::kRXDone, TsType::kPilotRX, frame_id),
-            demul_stats_.get_symbol_count(frame_id),
-            rx_counters_.num_pkts[prev_frame_slot],
-            rx_counters_.num_pkts_per_frame);
-    } break;
     case (PrintType::kPacketRXPilots):
-        printf("Main thread: received all pilots in frame: %zu  in %.2f us\n",
-            frame_id,
-            stats->master_get_delta_us(
+        printf("Main [frame %zu + %.1f ms]: Received all pilots\n", frame_id,
```

Comment:
The time to receive all pilots could be very short (less than 100 us) when using only one pilot in frequency orthogonal mode. We should print out more digits here.

Reply:
Agreed. I made all the timestamps to use two digits, which should get us 10-microsecond precision. I think going to 1-microsecond accuracy probably won't add much information since a `printf()` itself can take over a microsecond.